cmd/bnsd/app/app.go

```diff
@@ -63,10 +64,13 @@ func Router(authFn x.Authenticator, issuer weave.Address) app.Router {
 	// TODO: move to cash upon refactor
 	escrow.RegisterRoutes(r, authFn, namecoin.NewController())
 	multisig.RegisterRoutes(r, authFn)
-	blockchain.RegisterRoutes(r, authFn, issuer)
-	ticker.RegisterRoutes(r, authFn, issuer)
+	//TODO: Possibly revisit passing the bucket later to have more control over types?
+	// or implement a check
+	blockchain.RegisterRoutes(r, authFn, issuer, ticker.NewBucket().Bucket)
```

Comment:
pp: rather than passing Bucket, please have an interface in the handlers

Reply:
Yeah, like a Get interface